src/site/stages/build/plugins/process-entry-names.js

```diff
@@ -82,12 +82,16 @@ function processEntryNames(buildOptions) {
       if (!dom) continue;
 
       dom('script[data-entry-name],link[data-entry-name]').each((index, el) => {
+        // Derive the element properties.
         const $el = dom(el);
         const entryName = $el.data('entryName');
         const attribute = $el.is('script') ? 'src' : 'href';
-        const hashedEntryName = entryNamesDictionary.get(entryName);
-        const entryExists = files[hashedEntryName.slice(1)];
 
+        // Derive the hashed entry name.
+        const hashedEntryName = entryNamesDictionary.get(entryName) || [];
```

Comment:
:100:

Reply:
`hashedEntryName` will actually be a string, not an array.

Reply:
So, this could probably be recreated by -

```
npm run build -- --entry=does-not-exist
```

Since there isn't a `manifest.json` that has `does-not-exist` as its `entryName`, `hashedEntryName` would be `undefined`.  We could probably get clearer handling here knowing that. We could also probably incorporate the `buildOptions.entry` property, since without that argument, it should never happen.